shell/components/ResourceList/index.vue

```diff
@@ -142,18 +138,15 @@ export default {
       :type-display="customTypeDisplay"
       :schema="schema"
       :resource="resource"
+      :show-incremental-loading-indicator="showIncrementalLoadingIndicator"
+      :load-resources="loadResources"
+      :load-indeterminate="loadIndeterminate"
     >
-      <template v-slot:header>
-        <ResourceLoadingIndicator
-          v-if="showIncrementalLoadingIndicator"
-          :resources="loadResources"
-          :indeterminate="loadIndeterminate"
-        />
-      </template>
     </Masthead>
     <div v-if="hasListComponent">
       <component
         :is="listComponent"
+        :incremental-loading-indicator="showIncrementalLoadingIndicator"
```

Comment:
This is needed here because `showIncrementalLoadingIndicator` is a computed which cannot be overrun because of the async nature of getting the performance config.
The others are passed `v-bind="$data"` hence why they don't need to be passed along.

I tried to use these props on lists that have it's own Masthead, but I need to define as props on child, otherwise the child itself won't have access to them.

Unfortunately we need the plumbing as is 😢